CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.15.0] - 2022-02-22
+
+### Features
```

Comment:
The keyword to use is "added" (cf. changelog definitions at the beginning of the file).
Also, you need to update the git links at the end of the file.

Reply:
Ha ok i will make these changes. Thanks.